### PR TITLE
Sprint review: Initial configuration of the project

### PR DESCRIPTION
**Pull request by KyDev25**
Sprint review 1

### CODE REVIEW

File: sprint-reports/sprint-1.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+_Initial configuration so that everyone can start the project in a clean environment._
+
+## Positive reviews
+
+- Review 1
+
+  - Description: A dynamic, motivated team that does its utmost to give the best of itself.
+  - Commitment:
+    - Who: Barnabé
```
Comment (cedricgautier) [inline sprint-reports/sprint-1.md]: I think it would be clearer to say who said it instead of just saying "Who". I suggest to replace it by "Said by"

Comment (cedricgautier) [reply sprint-reports/sprint-1.md]:
```suggestion
    - Said by: Barnabé
```

**Review by BarnabePILLIAUDIN — APPROVED** [review]
LGTM

**Review by cedricgautier — APPROVED** [review]
You can merge tonight after trying to contact all participants. Thanks

**Pull request merged**